### ISSUE TITLE
realesrgan-ncnn-vulkan: can't compile

### ISSUE THREAD

**Issue by m911t** [NixOS/nixpkgs#203288]
### Describe the bug
Can't compile the program.

Log:
[report.txt](https://github.com/NixOS/nixpkgs/files/10098746/report.txt)

### Additional context
I'm using home-manager but I don't think that matters.

### Notify maintainers
@TilCreator 

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
 - system: `"x86_64-linux"`
 - host os: `Linux 6.0.8-zen1, NixOS, 23.05 (Stoat), 23.05pre429553.5dc7114b7b2`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.11.0`
 - channels(root): `"nixos"`
 - channels(m911t): `"home-manager"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by raphaelr** [comment]
Should be fixed by #201747 ([PR tracker](https://nixpk.gs/pr-tracker.html?pr=201747))

**Comment by m911t** [comment]
it is!
